src/transformers/utils/import_utils.py

```diff
@@ -754,11 +754,13 @@ def is_torch_xpu_available(check_device=False):
     if not is_torch_available():
         return False
 
-    import torch
-
+    torch_version = version.parse(_torch_version)
     if is_ipex_available():
         import intel_extension_for_pytorch  # noqa: F401
```

Comment:
Previously, if `is_ipex_available` evaluated to `False` we would return `False`. #31238 enables using pytorch for XPU as well, if the installed version >= 2.4. 

Here, we still return `False` if the installed version < 2.4 and IPEX is not available in the environment

Reply:
Thanks.

Could also be fixed by change `except RuntimeError` to `except RuntimeError, AttributeError`, but both works for me and leave you to make the call.

Reply:
I'll leave like this, as it makes the intended behaviour a bit more explicit imo :)